bookwyrm/views/books/books.py

```diff
@@ -90,6 +91,7 @@ def get(self, request, book_id, **kwargs):
             "rating": reviews.aggregate(Avg("rating"))["rating__avg"],
             "lists": lists,
             "update_error": kwargs.get("update_error", False),
+            "hyphenated_isbn13": hyphenator.hyphenate(book.isbn_13),
```

Comment:
I wonder if this would make more sense as a property on the book model? There are some generated-on-the-fly fields there already: https://github.com/bookwyrm-social/bookwyrm/blob/main/bookwyrm/models/book.py#L75

Reply:
Sure, I don't know where it makes the most sense but I don't mind changing that

Reply:
Yeah I think that makes a lot of sense